plugins/CoreAdminHome/Emails/TwoFactorAuthEnabledEmail.php

```diff
@@ -0,0 +1,21 @@
+<?php
+/**
+ * Matomo - free/libre analytics platform
+ *
+ * @link https://matomo.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ *
+ */
+
+namespace Piwik\Plugins\CoreAdminHome\Emails;
+
+use Piwik\Piwik;
+use Piwik\Plugins\CoreAdminHome\Emails\SecurityNotificationEmail;
+
+class TwoFactorAuthEnabledEmail extends SecurityNotificationEmail
+{
+    protected function getBody()
+    {
+        return Piwik::translate('CoreAdminHome_SecurityNotificationTwoFactorAuthEnabledBody');
+    }
+}
```

Comment:
For this email:

```
Someone enabled the two-factor authentication in your account. If it was you, carry on. If you don't recognize this activity, please reset your password.
```

If someone else enables two factor auth, the user will not be able to login anymore, so resetting their password might not be possible. Perhaps it should be, "email your superuser at ... and request they disable two factor auth and reset your password immediately"?

Reply:
should we use the first or all superuser email addresses for this?

Reply:
Not sure, we could have all but depending on how many there are that could be a lot of emails. With just one, the one superuser might get all of the requests. Do you have an opinion @tsteur?

Reply:
I modified the email messages, reusing from the password change email.